src/core/Jalasoft.TeamUp.Resumes.Core/ResumesService.cs

```diff
@@ -17,12 +17,12 @@ public ResumesService(IResumesRepository resumesRepository)
 
```

Comment:
Here you have to use IRepository as an Interface in the line 11 and 13

Reply:
And check startup.cs please it must use IRepository

Reply:
I change for IRepository in line 11 and 13.
I change in startup.cs the IRepository in line 22.